view/frontend/web/template/payment/form.html

```diff
@@ -139,47 +149,23 @@
                                maxlength="20" name="payment[customer_house_extension]"/>
                     </div>
                 </div>
-                <!-- ko if: !isAddressSameAsShipping() -->
-                <div class="field date required"
-                     data-bind="attr: {id: getCode() + '_delivery_address_street'}">
-                    <label data-bind="attr: {for: getCode() + '_delivery_address_street'}"
-                           class="label">
-                        <span><!-- ko i18n: 'Delivery street name'--><!-- /ko --></span>
-                    </label>
-                    <div class="control">
-                        <input type="text"
-                               data-bind="value: inputFields[getCode() + '_delivery_address_street']"
-                               data-validate="{required:true}"
-                               maxlength="100" name="payment[delivery_address_street]"/>
-                    </div>
-                </div>
-                <div class="field date required"
-                     data-bind="attr: {id: getCode() + '_delivery_address_housenumber'}">
-                    <label data-bind="attr: {for: getCode() + '_delivery_address_housenumber'}"
-                           class="label">
-                        <span><!-- ko i18n: 'Delivery house number'--><!-- /ko --></span>
-                    </label>
-                    <div class="control">
-                        <input type="text"
-                               data-bind="value: inputFields[getCode() + '_delivery_address_housenumber']"
-                               data-validate="{required:true}"
-                               maxlength="100" name="payment[delivery_address_housenumber]"/>
-                    </div>
+
+                <div class="field" data-bind="visible: toggle">
+                    <button class="action action-update" type="button" data-bind="click: updateAddress, click: toggleView">
+                        <span data-bind="i18n: 'Update'">Update</span>
+                    </button>
                 </div>
+
                 <div class="field date"
```

Comment:
@patrickcoppens In the above definitions the class "date" was removed. Shouldn't this one be removed as well?

Reply:
Done